### DISCUSSION TITLE
Did you train ArcFace and FaceNet yourself? Is it possible to additionally train already trained model. Not to train from the start.

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#867] [Q&A] [answered]
I find your model works the best compared to adaface and serengil/deepface ArcNet.

Did you train ArcFace and FaceNet yourself? 
Is it possible to additionally train already trained model. Not to train from the start, but "retrain" it a little bit?
Do you have any instructions on how to do it? 

What amount of RAM and GPU do I need?

**Comment by pospielov** [comment, marked as answer]
CompreFace uses the default ArcFace model from the InsightFace library. 
I'm not sure why our model works better than theirs. My guess - CompreFace reuses the code of InsightFace library. So our accuracy is the same as those researchers. And `serengil/deepface` uses custom Keras implementation. 
You can use instructions from InsightFace on how to train the MxNet model and put the new model into CompreFace.
You can "retrain it a little bit", and from my experience, Nvidia 3060 is enough for that, it's just very slow (may take a couple of days).
